src/sandbox/alternative-threading/FutureResponse.cs

```diff
@@ -48,7 +50,7 @@ public Response Response
 	                {
 	                    try
 						{
-							if (maxWait > 0) 
+							if (maxWait > TimeSpan.MinValue) 
```

Comment:
Not sure when this worked, but you can't compare a TimeSpan against 0 anymore. So I figured TimeSpan.Min is closest thing.

Reply:
Compare with TimeSpan.Zero instead.

Reply:
Good catch, fixed.